client/my-sites/hosting/cache-card/index.js

```diff
@@ -38,12 +115,18 @@ const CacheCard = ( {
 					primary
 					onClick={ clearCache }
 					busy={ isClearingCache }
-					disabled={ disabled || isClearingCache || shouldRateLimitCacheClear }
+					disabled={
+						disabled ||
+						isClearingCache ||
+						shouldRateLimitCacheClear ||
+						getEdgeCacheLoading ||
+						toggleEdgeCacheLoading
+					}
```

Comment:
Should we disable this button also when the cache is disabled?

Reply:
No, as when it's disabled, you can still clear the cache. Not the edge one, but the site's cache using [wp_cache_flush](https://developer.wordpress.org/reference/functions/wp_cache_flush/)